tests/vse_sync_pp/analyzers/test_ppsdpll.py

```diff
@@ -7,30 +7,35 @@
 from decimal import Decimal
 
 from vse_sync_pp.analyzers.ppsdpll import (
-    PhaseOffsetTimeErrorAnalyzer
+    TimeErrorAnalyzer
 )
 
 from .test_analyzer import AnalyzerTestBuilder
 
-DPO = namedtuple('DPO', ('phaseoffset',))
+DPLLS = namedtuple('DPLLS', ('timestamp','eecstate', 'phasestate', 'phaseoffset',))
```

Comment:
Your `TimeErrorAnalyzer` only requires `timestamp` and `phaseoffset`. Recommend removing `eecstate` and `phasestate` (if the parser changes these this analyzer doesn't care).

Reply:
From most recent review, this looks like it's moving in the right direction. Will open a new comment on next review if needed.